bokehjs/src/coffee/common/ui_events.coffee

```diff
@@ -88,11 +86,30 @@ class UIEvents extends Backbone.Model
       logger.debug("Registering tool: #{type} for event 'doubletap'")
       tool_view.listenTo(@, "doubletap", tool_view._doubletap)
 
+    # Dual touch hack part 1/2
+    # This is a hack for laptops with touch screen who may be pinching or scrolling
+    # in order to use the wheel zoom tool. If it's a touch screen the WheelZoomTool event
+    # will be linked to pinch. But we also want to trigger in the case of a scroll.
+    if 'ontouchstart' of window or navigator.maxTouchPoints
```

Comment:
Shouldn't this line contain `> 0` as in the other cases? Perhaps an utility function could be added to avoid duplication of logic.